{"timestamp":"2025-07-11T21:27:23.236236","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-07-11T21:27:23.237162","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/test1.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-07-11T21:27:23.264626","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"UniqueViolation","exc_value":"duplicate key value violates unique constraint \"service_pkey\"\nDETAIL:  Key (id)=(1) already exists.\n","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":867,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1159,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/test1.py","lineno":74,"name":"generar_servicios_dispositivos"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-07-11T21:27:23.327234Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:27:23.327434Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:27:23.327547Z","level":"info","event":"Current task name:generar_servicios","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:27:23.327683Z","level":"info","event":"Dag name:cargar_datos","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:27:23.327808Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:27:23.327954Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:27:23.328113Z","level":"info","event":"Task:<Task(PythonOperator): generar_servicios>","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:27:23.328265Z","level":"info","event":"Failure caused by duplicate key value violates unique constraint \"service_pkey\"","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:27:23.328419Z","level":"info","event":"DETAIL:  Key (id)=(1) already exists.","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:27:23.328570Z","level":"info","event":"","chan":"stdout","logger":"task"}
